src/misc/messages.cpp

```diff
@@ -145,7 +205,7 @@ static bool LoadMessageFile(const std_fs::path &filename)
 			// This second if should not be needed, but better be safe.
 			if (ll && message[ll - 1] == '\n')
 				message[ll - 1] = 0;
-			MSG_Replace(name, message);
+			MSG_Replace(name, message, is_utf8);
```

Comment:
Well, okay, maybe if this is a file-local function, and you're always passing the flag in via a well-named variable like you're doing it here, then I think the boolean can stay.

But if you're making any of these functions part of the public API, I still recommend to ditch the flag and create two versions.

Reply:
This is actually used internally only, changed to static function.